index.html

```diff
@@ -0,0 +1,365 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Youtube</title>
+    <link rel="stylesheet" href="style.css">
+    <script
+    src="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/js/all.min.js"
+    defer
+  ></script>
+</head>
+<body>
+  <header>
+    <div>
+        <img src="./img/menu-button.png" alt="menu icon button" id="menu-icon">
+    </div>
+
+        <a href="https://youtube.com/"></a><img src="./img/yt-logo.jpg" alt="youtube logo" id="yt-logo"></a>
+
+    <div id="search-container">
+        <input type="text" placeholder="Search..." id="search-bar">
+        <button id="search-button">
+            <img src="./img/search-icon.jpg" alt="search icon" id="search-icon">
+        </button>
+    </div>
+
+    <button class="topright"><img id="upload" class="header-image" src="./img/upload-vid.jpg" alt="upload video"></button>
+
+    <button class="topright"><img id="ytapps" class="header-image" src="./img/ytapp.png" alt="youtube apps"></button>
+
+    <button class="topright"><img id="options" class="header-image" src="./img/ellipsis.png" alt="options"></button>
+      
+    <button id="sign-in_btn">Sign in</button>
```

Comment:
Here, you have an overuse of the `id` attribute. The purpose of which you have them can be interchanged with the `class `attribute. The id attribute is used only when it is needed and its use is very specific. If you want to add a unique identifier to an element, you can do that by adding multiple classes, give one of them a unique name and use that class name for styling. Kindly, remove the `id` attribute and implement the suggestion above as they are not needed.